siriuspy/tests/bsmp/test_commands.py

```diff
@@ -2,42 +2,50 @@
 
 """Test commands module."""
 
+import struct
 from unittest import TestCase
 from unittest.mock import Mock
-import struct
-
-from siriuspy.bsmp import Package, Message, Types, Variable, VariablesGroup, \
-    Function, BSMP, SerialAnomResp
 
+from siriuspy.bsmp import (
+    BSMP,
+    Function,
+    Message,
+    Package,
+    SerialAnomResp,
+    Types,
+    Variable,
+    VariablesGroup,
+)
 from siriuspy.util import check_public_interface_namespace
 
 
 class TestBSMPAPI(TestCase):
     """Test BSMP."""
 
     api = (
-        'entities', 'channel',
-        'query_protocol_version',
-        'query_list_of_variables',
-        'query_list_of_group_of_variables',
-        'query_group_of_variables',
-        'query_list_of_curves',
-        'query_curve_checksum',
-        'query_list_of_functions',
-        'read_variable',
-        'read_group_of_variables',
-        'write_variable',
-        'write_group_of_variables',
-        'binoperation_variable',
-        'binoperation_group',
-        'write_and_read_variable',
-        'create_group_of_variables',
-        'remove_all_groups_of_variables',
-        'request_curve_block',
-        'curve_block',
-        'recalculate_curve_checksum',
-        'execute_function',
-        'anomalous_response',
+        "entities",
```

Comment:
@carneirofc, why double quotes? Single quotes are prettier, aren't they?

Reply:
I will roll back these changes. This ended up happening due to the linter/"code formatter" I use.
The double quotes is just an influence of other languages and also happens to be the default option on my linter of choice.

Reply:
ok @carneirofc, thanks!